test/shjs.js

```diff
@@ -0,0 +1,19 @@
+var shell = require('..');
+var path = require('path');
+var assert = require('assert');
+
+function runScript(name) {
+  return shell.exec(path.resolve(__dirname, '../bin/shjs') +
+                    ' ' +
+                    path.resolve(__dirname, 'resources', 'shjs', name), { silent: true });
+}
+
+assert(runScript('exit-codes.js'), 42, 'exit code works');
+
+var stdioRet = runScript('stdout-stderr.js');
+assert(stdioRet.stdout, 'stdout: OK!\n', 'stdout works');
+assert(stdioRet.stderr, 'stderr: OK!\n', 'stderr works');
+
+assert(runScript('coffeescript.coffee').stdout, 'CoffeeScript: OK!\n');
```

Comment:
assert.equal, like above. Also, perhaps we could check its error code to make sure it's 0?